src/runtime_src/core/edge/drm/zocl/zocl_bo.c

```diff
@@ -202,7 +202,8 @@ zocl_create_bo(struct drm_device *dev, uint64_t unaligned_size, u32 user_flags)
 		 * not establish the kernel mapping. We just can not
 		 * access BO directly from kernel.
 		 */
-		bo->vmapping = memremap(bo->mm_node->start, size, MEMREMAP_WC);
+		if (zdev->mem[bank].zm_type != ZOCL_MEM_TYPE_LPDDR)
```

Comment:
Any specific reason we can not map a virtual address on LPDDR? Without this, xclBOWrite/xclBORead won't work

Reply:
> Please help me understand why we can not set up a virtual mapping of LPDDR.
> Other than that, if we can treat LPDDR and PLDDR equally, I don't think it is necessary to separate ZOCL_MEM_TYPE_PLDDR and ZOCL_MEM_TYPE_LPDDR. We can find one better name cover these two memory type.
> Otherwise, please search all code which is using PLDDR and make enhancement according, e.g. zocl_update_mem_stat, zocl_create_bo, etc.

Hi @larry9523 , We should have separate tags for LPDDR and PLDDR as they  are two different types of memories. For example, we need to have cache enabled for LPDDR. we need a clear differentiation.
@saifuddin-xilinx , Please search the PLDDR in the code and make necessary changes

Reply:
@chvamshi-xilinx separate tag is OK. I am saying we can unify the behavior on both tags. The only difference I see is we are not doing memremap on LPDDR, which I don't quite understand why.

Reply:
Yes @larry9523 , I Agree. @saifuddin-xilinx , We need to debug why memremap is not working. We also need to enable Cache for LPDDR.

Reply:
After changing in dtsi file for LPDDR region as "no-map" the memremap issue resolved. Here I have updated zocl code for LPDDR case. And will send a separate PR for plarform changes.